### ISSUE TITLE
pom.xml unresolved dependencies

### ISSUE THREAD

**Issue by elimirks** [ClearVolume/imglib2-clearvolume#5]
When compiling, I am getting an error while retrieving net.clearvolume:clearvolume:jar:1.0-SNAPSHOT.

Here is the full error message:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 11.237 s
[INFO] Finished at: 2016-03-01T14:24:45-05:00
[INFO] Final Memory: 19M/233M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project imglib-clearvolume: Could not resolve dependencies for project sc.fiji:imglib-
clearvolume:jar:1.1.0: The following artifacts could not be resolved: net.clearvolume:clearvolume:jar:1.0-SNAPSHOT, net.
clearvolume:cleargl:jar:1.0-SNAPSHOT, net.clearvolume:clearcuda:jar:0.9-SNAPSHOT: Could not find artifact net.clearvolum
e:clearvolume:jar:1.0-SNAPSHOT in imagej.public (http://maven.imagej.net/content/groups/public) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

The repository seems to have been moved http://maven.imagej.net/content/groups/public/net/

*Update 1:*

It seems to be available here: http://dl.bintray.com/clearvolume/ClearVolume/net/clearvolume/

I found that url in the pom.xml, so maybe it is just a configuration error. I will keep looking into this

*Update 2:*

Turns out this is only a problem with compiling in development mode: `mvn` or `mvn -P development`. Running `mvn -P production` compiles nicely

**Comment by royerloic** [comment]
I think you need to add the ClearVolume repositories in your POM (not just
the dependencies)
Here is the gradle equivalent:

repositories {
    maven { url  "http://dl.bintray.com/clearvolume/ClearVolume" }
    maven { url  "http://dl.bintray.com/rtlib/CoreMem" }
}

Hope this helps!



On Tue, Mar 1, 2016 at 8:33 PM, Elijah Mirecki <notifications@github.com>
wrote:

> When compiling, I am getting an error while retrieving
> net.clearvolume:clearvolume:jar:1.0-SNAPSHOT.
>
> Here is the full error message:
>
> [INFO] ------------------------------------------------------------------------
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time: 11.237 s
> [INFO] Finished at: 2016-03-01T14:24:45-05:00
> [INFO] Final Memory: 19M/233M
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal on project imglib-clearvolume: Could not resolve dependencies for project sc.fiji:imglib-
> clearvolume:jar:1.1.0: The following artifacts could not be resolved: net.clearvolume:clearvolume:jar:1.0-SNAPSHOT, net.
> clearvolume:cleargl:jar:1.0-SNAPSHOT, net.clearvolume:clearcuda:jar:0.9-SNAPSHOT: Could not find artifact net.clearvolum
> e:clearvolume:jar:1.0-SNAPSHOT in imagej.public (http://maven.imagej.net/content/groups/public) -> [Help 1]
> [ERROR]
> [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> [ERROR] Re-run Maven using the -X switch to enable full debug logging.
> [ERROR]
> [ERROR] For more information about the errors and possible solutions, please read the following articles:
> [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
>
> The repository seems to have been moved
> http://maven.imagej.net/content/groups/public/net/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ClearVolume/imglib2-clearvolume/issues/5>.
>



-- 
Dr. Loic Royer
Bischofsweg 44
01099 Dresden
Germany

Phone:
 cellphone:  +49 157 84900128

Internet:
 email:       royerloic@gmail.com
 skype:      royerloic

**Comment by skalarproduktraum** [comment]
This is fixed with #8, sorry for the inconvenience!
